src/components/app/EnterpriseAppPageRoutes.jsx

```diff
@@ -16,27 +17,27 @@ import { PathwayProgressPage } from '../pathway-progress';
 // to reduce API calls by 2 (DashboardPage, CoursePage, SearchPage) or by 3 ( + AuthenticatedPage) if created in App.jsx
 const EnterpriseAppPageRoutes = () => (
   <AuthenticatedUserSubsidyPage>
-    <PageRoute exact path="/:enterpriseSlug" component={DashboardPage} />
-    <PageRoute
-      exact
-      path={['/:enterpriseSlug/search', '/:enterpriseSlug/search/:pathwayUUID']}
-      component={SearchPage}
-    />
-    <PageRoute exact path="/:enterpriseSlug/course/:courseKey" component={CoursePage} />
-    <PageRoute path="/:enterpriseSlug/:courseType/course/:courseKey" component={CoursePage} />
-    {features.ENABLE_PROGRAMS && (
-      <PageRoute exact path="/:enterpriseSlug/program/:programUuid" component={ProgramPage} />
-    )}
-    {
-      // Deprecated URL, will be removed in the future.
-      <PageRoute exact path="/:enterpriseSlug/program-progress/:programUUID" component={ProgramProgressRedirect} />
-    }
-    <PageRoute exact path="/:enterpriseSlug/program/:programUUID/progress" component={ProgramProgressPage} />
-    <PageRoute exact path="/:enterpriseSlug/skills-quiz" component={SkillsQuizPage} />
-    <PageRoute exact path="/:enterpriseSlug/licenses/:activationKey/activate" component={LicenseActivationPage} />
-    {features.FEATURE_ENABLE_PATHWAY_PROGRESS && (
-      <PageRoute exact path="/:enterpriseSlug/pathway/:pathwayUUID/progress" component={PathwayProgressPage} />
-    )}
+    <Routes>
+      <Route path="/" element={<PageWrap><DashboardPage /></PageWrap>} />
+      {['search', 'search/:pathwayUUID'].map(route => (
+        <Route
```

Comment:
I think we might need a "key" prop here to avoid the `Warning: Each child in a list should have a unique "key" prop.` error:
```
 {['search', 'search/:pathwayUUID'].map(route => (
        <Route
          key={route}
          path={route}
          element={<PageWrap><SearchPage /></PageWrap>}
        />
      ))}
      ```